### ISSUE TITLE
Sort cam order?

### ISSUE THREAD

**Issue by angusmedia** [ccrisan/motioneye#296]
It would be nice to be able to sort the order of the cams. I always have to erase all cams and readd them in the right order wether I want a special cam to be first.

**Comment by chrisf4lc0n** [comment]
No you don't, just find the locations of thread-*.conf files and change numbers to the order you want them to be visible in MotionEye.
1st camera thread-1.conf
2nd camera thread-2.conf
etc...

**Comment by ioquatix** [comment]
I too would like this feature.

**Comment by harolds** [comment]
Yep rearranging is possible by first stopping both motioneye and motion:
sudo service motioneye stop
sudo service motion stop

Then rename the files:
mv thread-1.conf backup.conf
mv thread-5.conf thread-1.conf
mv backup.conf thread-5.conf

And restart everything:
sudo service motioneye start
sudo service motion start

**Comment by hasselk** [comment]
thanks harold thats a good point, but maybe this should be a feature in motioneye
1. the possibility to rename/rearange the files/id and anything that belongs to it like the created folder in /var/lib/motioneye
2. but even better would be the added possibility to rearrange cams while viewing and save that as a layout which can be reloaded

**Comment by harolds** [comment]
Your second point should be the easiest to implement. Could even be stored in a cookie. Or/and stored with an account. 

Groeten,
Harold.

**Comment by SteveRMann** [comment]
A year later- no progress on this feature?

**Comment by sticilface** [comment]
I to would like this as my system evolves it is rather challenging to rearrange things

**Comment by shaynethiessen** [comment]
It would be nice if this was a feature :)
